Installing-Oppia-(Windows;-Python-3).md

```diff
@@ -160,15 +160,15 @@ You will be asked about the setup for the panel. Just click on default config.
 
 ![image](https://user-images.githubusercontent.com/23002114/94156539-d24fff00-fe9d-11ea-8f7a-039db7a7ce51.png)
 
-## Run the E2E tests
+## Step 9: Run Oppia locally
```

Comment:
This file's table of contents needs to be updated too

Reply:
Done